docs/high-level-overview.md

```diff
@@ -51,6 +51,8 @@ Object      | Examples   | Linked Objects | State | Possible Attributes
 
 *Table1:* Shows the relation of Objects to other Objects and Attributes possible in each state.
 
+![Core Objects](img/core-objects.png)
```

Comment:
This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.